recipes/lbfgspp/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "lbfgspp" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/yixuan/lbfgspp/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 7101744a538c3aff52e10c82267305847b0b5e9d39f9974b4b29812cd1398ff9
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - eigen >=3,<4
+  run:
+    - eigen >=3,<4
+
+test:
+  source_files:
+    - examples/
+  requires:
+    - {{ compiler('cxx') }}
+  commands:
+    - test -f $PREFIX/include/LBFGS.h  # [unix]
+    - test -f $PREFIX/include/LBFGSB.h  # [unix]
+    - test -d $PREFIX/include/LBFGSpp  # [unix]
+    - if not exist %LIBRARY_PREFIX%\include\LBFGS.h exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\include\LBFGSB.h exit 1  # [win]
+    - for i in examples/*.cpp; do $CXX $CXXFLAGS -o "$i.out" "$i" && "$i.out"; done  # [unix]
```

```suggestion
    - for i in examples/*.cpp; do $CXX $CXXFLAGS -I$PREFIX/include/eigen3 -o "$i.out" "$i" && "$i.out"; done  # [unix]
```
This is probably the issue with your script.